Shared/Models/NetworkAnalyticsEvents.swift

```diff
@@ -191,4 +209,25 @@ struct DownloadErrorEvent: DownloadAnalyticsEventPayload {
     }
 }
 
+struct SuccessBackupEvent: BackupEventPayload {
+    var eventName = NetworkAnalyticsEvent.SUCCESS_BACKUP
+    var foldersToBackup: Int
+    func getProperties() -> [String : Any] {
+        return [:]
```

Comment:
We are missing a property here, in the dictionary, you can call it: `folders_to_backup_count` and add the `foldersToBackup` value